### PR TITLE
release sonar-findbugs 3.11.1 for latest SQ

### PR DESCRIPTION
**Pull request by KengoTODA**
This is the latest version that works with SpotBugs 3.1.12.
* [Changelog](https://github.com/spotbugs/sonar-findbugs/releases/tag/3.11.1)
* [Download](https://github.com/spotbugs/sonar-findbugs/releases/download/3.11.1/sonar-findbugs-plugin-3.11.1.jar)
* [SonarCloud](https://sonarcloud.io/dashboard?branch=3.11.1&id=com.github.spotbugs%3Asonar-findbugs-plugin)

### CODE REVIEW

**Comment by KengoTODA** [comment]
Here is [the post in community forum](https://community.sonarsource.com/t/new-release-sonar-findbugs-3-11-1/14580), please check.

File: findbugs.properties [hunk 1]
```diff
@@ -1,11 +1,17 @@
...
 defaults.mavenArtifactId=sonar-findbugs-plugin
 
+3.11.1.description=Use SpotBugs 3.1.12
+3.11.1.sqVersions=[7.6,LATEST]
+3.11.1.date=2019-09-23
+3.11.1.changelogUrl=https://github.com/spotbugs/sonar-findbugs/releases/tag/3.11.1
+3.11.1.downloadUrl=https://github.com/spotbugs/sonar-findbugs/releases/download/3.11.1/sonar-findbugs-plugin-3.11.1.jar
+
 3.11.0.description=Use SpotBugs 3.1.12
 3.11.0.sqVersions=[7.6,LATEST]
```
Comment (ganncamp) [inline findbugs.properties]: Only one version can be compatible with `LATEST`. The convention is to edit the old version's range end to the SQ version current at the time.

File: findbugs.properties [hunk 2]
```diff
@@ -1,11 +1,17 @@
...
 description=Provide Findbugs rules for analysis of Java projects
-archivedVersions=2.4,3.0,3.1,3.2,3.3,3.4.3,3.4.4,3.5,3.6,3.7,3.8.0,3.9.0,3.9.1,3.9.2,3.9.3,3.10.0
-publicVersions=3.9.4,3.11.0
+archivedVersions=2.4,3.0,3.1,3.2,3.3,3.4.3,3.4.4,3.5,3.6,3.7,3.8.0,3.9.0,3.9.1,3.9.2,3.9.3,3.10.0,3.11.0
+publicVersions=3.9.4,3.11.1
 
 defaults.mavenGroupId=org.sonarsource.sonar-findbugs-plugin
 defaults.mavenArtifactId=sonar-findbugs-plugin
 
+3.11.1.description=Use SpotBugs 3.1.12
```
Comment (ganncamp) [inline findbugs.properties]: Did you copy/paste and forget to update the description?

Comment (KengoTODA) [reply findbugs.properties]: no the version of spotbugs hasn't been changed.

**Comment by KengoTODA** [comment]
resolved, please check

File: findbugs.properties (hunk 2 above)
Comment (ganncamp) [reply findbugs.properties]: Your description here does not match the description on the release. As a user if I look at your current description I will be far less motivated to upgrade than if I see "Fix CVE-2019-10173". _That_ should be the description for this version.

Comment (KengoTODA) [reply findbugs.properties]: updated.

**Pull request merged**